frontend/public/components/cluster-overview.jsx

```diff
@@ -214,20 +216,20 @@ const permissionedLoader = () => {
     );
 };
 
-export const ClusterOverviewPage = props => {
+const ClusterOverviewPage_ = props => {
+  const { OPENSHIFT: openshiftFlag, PROJECTS_AVAILABLE: projectsFlag } = props.flags;
+  const fake = !flagPending(openshiftFlag) && !flagPending(projectsFlag) && openshiftFlag && !projectsFlag;
   const namespace = _.get(props, 'match.params.ns');
-  let title = 'Cluster Status';
-  if (namespace) {
-    title = `Status of ${ namespace }`;
-  }
+  const title = namespace ? `Status of ${ namespace }` : 'Cluster Status';
+
   return <React.Fragment>
     <StartGuide dismissible={true} style={{margin: 15}} />
     <Helmet>
-      <title>{title}</title>
+      <title>{fake ? 'Overview' : title}</title>
     </Helmet>
-    <NavTitle title={title} />
+    <NavTitle title={fake ? 'Overview' : title} />
     <div className="cluster-overview-cell container-fluid">
-      <AsyncComponent namespace={namespace} loader={permissionedLoader} />
+      <AsyncComponent namespace={namespace} loader={permissionedLoader} openshiftFlag={openshiftFlag} fake={fake} />
```

Comment:
Does `openshiftFlag` need to be passed through since it's already connected to `GraphsPage`?

Reply:
GraphsPage isn't connected to the flags any more. I hoisted that up and connected the flags to ClusterOverviewPage because I needed them there to change the title to 'Overview'. If we could come up with a better default name that could be used for any context, I could just connect the GraphPage component instead.

Reply:
OK thanks, I see that now